webhook/resourcesemantics/conversion/conversion.go

```diff
@@ -109,17 +110,26 @@ func (r *reconciler) convert(
 	out := outZygote.DeepCopyObject().(ConvertibleObject)
 
 	hubGVK := inGVK.GroupKind().WithVersion(conv.HubVersion)
-	ctx = logging.WithLogger(ctx, logger.With(
+
+	logger = logger.With(
 		zap.String("inputType", formatGVK(inGVK)),
 		zap.String("outputType", formatGVK(outGVK)),
 		zap.String("hubType", formatGVK(hubGVK)),
-	))
+	)
 
 	// TODO(dprotaso) - potentially error on unknown fields
 	if err = json.Unmarshal(inRaw.Raw, &in); err != nil {
 		return ret, fmt.Errorf("unable to unmarshal input: %w", err)
 	}
 
+	if acc, err := kmeta.DeletionHandlingAccessor(in); err == nil {
+		logger = logger.With("key", acc.GetNamespace()+"/"+acc.GetName())
```

Comment:
note our controllers have a private `safeKey` method in case they're operating against clustered scope objects

Reply:
I don't think knative resources (which are only ones here) can be cluster scoped?

Reply:
Went the fancy way :)